.github/workflows/aws_deploy.yml

```
@@ -148,6 +148,21 @@ jobs:
           chmod +x ./.github/scripts/update_ecs.sh
           ./.github/scripts/update_ecs.sh
 
+      - name: Run Database Migrations
+        env:
+          TURSO_DATABASE_URL: ${{ secrets.TURSO_DATABASE_URL }}
+          TURSO_AUTH_TOKEN: ${{ secrets.TURSO_AUTH_TOKEN }}
+        run: |
+          # Install Turso CLI
+          curl -sSfL https://get.tur.so/install.sh | bash
+          export PATH="$HOME/.turso:$PATH"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)